Configuration/Skimming/python/autoSkim.py

```diff
@@ -32,6 +34,8 @@
     'Mu' : 'WMu+ZMu+HighMET+LogError',    
     'EG':'WElectron+ZElectron+HighMET+LogError',
     'TopMuEG':'TopMuEG+LogError',
+    'MuonPOG': 'MuonPOGSkim+LogError',
+    'MuonJPsiPOG': 'MuonPOGJPsiSkim+LogError',
```

Comment:
I'm curious why these are only in ```autoSkimRunI ``` 
I'm not well aware of the use cases though.

Reply:
I think these two lines should be removed. Also we do not have a mapping of MuonPOG/MuonJPsiPOG PD.

Reply:
Hi, I've removed those lines from autoSkim. @kfjack can you point me have to do the mapping?

Reply:
I assume MuonPOGSkim skim should run on single Mu PD, and MuonPOGJPsiSkim should run on Charmonium PD, right? You can just add the skim to 
'SingleMuon' : 'ZMu+MuTau+LogError+LogErrorMonitor',
'Charmonium' : 'BPHSkim+LogError+LogErrorMonitor',
e.g. 
'SingleMuon' : 'MuonPOGSkim+ZMu+MuTau+LogError+LogErrorMonitor',
'Charmonium' : 'MuonPOGJPsiSkim+BPHSkim+LogError+LogErrorMonitor',